modules/docs/arrow-docs/docs/docs/arrow/data/eithert/README.md

```diff
@@ -166,18 +172,18 @@ import arrow.fx.rx2.extensions.fx
 
 fun getCountryCode(personId: Int): ObservableK<Either<BizError, String>> =
   ObservableK.fx {
-    val person = findPerson(personId).bind()
-    val address = person.fold (
+    val person = !findPerson(personId)
```

Comment:
are we removing `.bind()` ? I actually liked it more than `!`, never been super convinced about overriding that operator tbh.

Reply:
No we're allowing different styles. We still have other areas in the docs where we use bind, 
But I don't mind changing it back to `.bind()`

Reply:
I guess it's a longer discussion we need to have outside of this PR. We should probable agree on one version and go with it for everything (including docs) cc @pakoito @nomisRev @raulraja. As said, not blocker for this PR. For now I guess you can keep it as is.